targets/csharp/templates/Errors.cs.ejs

```diff
@@ -47,7 +47,10 @@ namespace PlayFab
     public class PlayFabBaseResult
     {
         public object CustomData;
-        public PlayFabError Error;
```

Comment:
Why was it problematic to have the PlayFabError type directly in the base result? What does this mean for the future of the PlayFabError type?

Duplicating the contents of PlayFabError into here seems to introduce some ambiguity as to the purpose of each as well as increases code maintenance.